cmd/galaxy_install.go

```diff
@@ -45,15 +50,45 @@ func (c *GalaxyInstallCommand) Run(args []string) int {
 		c.UI.Warn("Warning: multiple role files found. Defaulting to galaxy.yml")
 	}
 
+	mockUi := cli.NewMockUi()
+
 	galaxyInstall := execCommand("ansible-galaxy", "install", "-r", files[0])
-	logCmd(galaxyInstall, c.UI, true)
+	logCmd(galaxyInstall, mockUi, true)
 	err := galaxyInstall.Run()
 
 	if err != nil {
 		c.UI.Error(fmt.Sprintf("Error running ansible-galaxy: %s", err))
```

Comment:
We should print the original stderr here because:

Using a9a1ac8 plus a typo in `galaxy.yml`:

```sh-session
$ ansible-galaxy install -r galaxy.yml
- composer (1.7.6) is already installed, skipping.
- ntp (1.6.4) is already installed, skipping.
- logrotate (v0.0.5) is already installed, skipping.
- swapfile (v2.0.22) is already installed, skipping.
- mailhog (2.1.4) is already installed, skipping.
- trellis-backup-during-deploy (0.1.2) is already installed, skipping.
- TypistTech.trellis-newrelic-php (master) is already installed, skipping.
- downloading role 'xxxxxxxxxxxxxtrellis-cloudflare-origin-ca', owned by TypistTech
 [WARNING]: - TypistTech.xxxxxxxxxxxxxtrellis-cloudflare-origin-ca was NOT
installed successfully: - sorry, TypistTech.xxxxxxxxxxxxxtrellis-cloudflare-
origin-ca was not found on https://galaxy.ansible.com.

ERROR! - you can use --ignore-errors to skip failed roles and finish processing the list.


$ /Users/rufus/Code/trellis-cli/trellis-cli galaxy install
Running command => ansible-galaxy install -r galaxy.yml
Error running ansible-galaxy: exit status 1
```

Reply:
Yeah good point 👍 done

Reply:
`c.UI.Error(mockUi.ErrorWriter.String())` should go before line 60 (inside the `if`)
.